hikari/applications.py

```diff
@@ -476,8 +476,11 @@ class Application(guilds.PartialApplication):
     Will be `builtins.None` if this application doesn't have a bot.
     """
 
-    owner: users.User = attr.ib(eq=False, hash=False, repr=True)
-    """The application's owner."""
+    owner: typing.Optional[users.User] = attr.ib(eq=False, hash=False, repr=True)
+    """The application's owner.
+
+    Will be `builtins.None` if the application is owned by Discord.
```

Comment:
Is this ever actually the case or is this field just not included in this place for privacy reasons. (Do we really want to muddle up the interface of main objects to cover cases where a partial representation could also work better with a lot less fields)

Reply:
This field seems to be included in every single application except for the embedded ones. The only one I have tested this against is YouTube Together, which seems to be implemented by Discord.

I wasn't sure how to implement this without having to create a separate object that might break in the future.

Reply:
This field is only included in two of the application models and is rarely included in partial representations though

Reply:
Also making this object cover more cases than just full application representations will be just as likely to be broken down the line